pyproject.toml

```diff
@@ -1,14 +1,16 @@
 [build-system]
-requires = ["setuptools>=64", "setuptools-scm>=8"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
 
 [project]
 name = "bagit"
 dynamic = ["version"]
 description = "Create and validate BagIt packages"
 readme = {file = "README.rst", content-type = "text/x-rst"}
 authors = [
-    { name = "Ed Summers", email = "ehs@pobox.com" },
```

Comment:
Andy wrote much of the original code, and Chris has added to it since I last touched it, so this seems right?

Reply:
I guess that's an interesting question: do we try to list everyone who's made significant contributions in the past, or only the people who want to get emails when someone needs help?